src/cpp/rtps/transport/tcp/RTCPMessageManager.cpp

```diff
@@ -472,6 +472,10 @@ ResponseCode RTCPMessageManager::processBindConnectionRequest(
 
     sendData(channel, BIND_CONNECTION_RESPONSE, transaction_id, &payload, code);
 
+    // Add pending logical ports to the channel
+    mTransport->send_channel_pending_logical_ports(channel->locator(), channel);
```

Comment:
If we are always using the channel's locator, we might remove the first argument from the method.